scrapy/extensions/feedexport.py

```diff
@@ -151,7 +151,7 @@ def __init__(self, settings):
         if not self._exporter_supported(self.format):
             raise NotConfigured
         self.store_empty = settings.getbool('FEED_STORE_EMPTY')
-        self.export_fields = settings.getlist('FEED_EXPORT_FIELDS')
+        self.export_fields = settings.getlist('FEED_EXPORT_FIELDS') or None
```

Comment:
So FEED_EXPORT_FIELDS can't be set to `[]` by the user with this change, right? I think it makes sense, not sure what's the actual use-case of that.

Maybe we should fix this in Settings, but I think the only way of knowing if the actual value was set as `[]` or isn't set at all is by checking with `settings.get()` before.